frontend/web/components/modals/Rule.js

```diff
@@ -16,6 +15,14 @@ export default class Rule extends PureComponent {
         const operatorObj = Utils.findOperator(rule.operator, rule.value, operators);
         const operator = operatorObj && operatorObj.value;
         const value = typeof rule.value === 'string' ? rule.value.replace((operatorObj && operatorObj.append) || '', '') : rule.value;
+        let valuePlaceholder = 'Value *';
+        if (operator) {
+            if (operator === 'MODULO') {
+                valuePlaceholder = 'Value(<divisor>|<remainder>) *';
```

Comment:
I appreciate this was someone else's PR. We should try and keep all this as configurable as possible, lets add in add an optional valuePlaceholder to the segment rules remote config. Then lines 18-25 can just be:

```
valuePlaceholder = rule.hideValue?  "Value (N/A)" : rule.valuePlaceholder || "Value"
```